### ISSUE TITLE
Allow to mock exception

### ISSUE THREAD

**Issue by VladislavAntonyuk** [Apps72/DbMocker#20]
I would like to return custom exception (TimeoutException, Win32Exception etc)
so I can write:
```csharp
conn.Mocks
                .When(cmd => cmd.CommandText.Contains(sql))
                .ThrowException(expectedException);
```

**Comment by dvoituron** [comment]
That's a very good idea.
I will add this feature for the next release. 

Thanks

**Comment by wessonben** [comment]
I vote for this feature too. I looked for exactly this today....

```
            MockDbConnection.Mocks
                .WhenAny()
                .Throws(new Exception("Example database error."));
```

**Comment by wessonben** [comment]
Just submitted a PR for a couple of simple methods to throw exceptions. Hope you find it useful.

**Comment by dvoituron** [comment]
Hello, I merged your PR in the **master** branch. 
I renamed the method to 'ThrowsException' to avoid misunderstandings and future evolutions of the code.
The package is published in v 1.18: https://www.nuget.org/packages/DbMocker/1.18.0
